packages/haul-core/src/server/NonInteractiveUI.ts

```diff
@@ -0,0 +1,33 @@
+import Runtime from '../runtime/Runtime';
+
+export default class NonInteractiveUserInterface {
```

Comment:
Maybe sharing a common interface between `InteractiveUI` and `NonInteractiveUI` makes sense? Unless TS infers it somehow already, then it's not needed

Reply:
I forgot to add `implemented UserInterface` as in `InteractiveUI`